### PR TITLE
Fix #343: Add Export of All Participants

### LINKED ISSUE #343: Participants page: add export of all participants.
From the 2019-02-04 meeting at SWOP with Chris and Jessica:

On the Home->Participants page, add the ability to export all participants.  Include all fields per participant, not just the fields that happen to be displayed on the Participants page.

### PR DESCRIPTION
**Pull request by frankduncan**
Added a mixin for different Admin pages to be able to export, and enabled for the ParticipantAdmin.

Decided to go the mixin route in case someone asks for export of other admin interfaces, this makes those changes a simple mixin addition.

Need #347 to land before you can test this.

### CODE REVIEW

File: streetcrm/admin.py [hunk 1]
```diff
@@ -791,6 +791,17 @@ class ParticipantAdmin(mixins.AdminArchiveMixin, mixins.SignInSheetAdminMixin, S
     )
     import_path = reverse_lazy("import-participants")
 
+    export_header = [
```
Comment (kfogel) [inline streetcrm/admin.py]: So if we add more fields to Participant, we'll have to remember to come here and update this list, right?

If so, could we at least add a comment wherever Participant is defined in the model, mentioning this, and a cross-referencing comment here.

Comment (frankduncan) [reply streetcrm/admin.py]: Well, yes and no.  This isn't the full list of everything that's actually in participant.  It's the list of what was in the export when you do a search for participants (for instance, city is omitted).  Maybe we should step back and do a clarification of what should be in the export?

Comment (kfogel) [reply streetcrm/admin.py]: What the users requested was that when exporting Participants, we export everything about them, including even the internal ID.  This makes sense to me -- they can always ignore a column in the CSV if they want to, but they can't add a column that's not there.  As stated in issue #343: "Include all fields per participant, not just the fields that happen to be displayed on the Participants page."

Comment (frankduncan) [reply streetcrm/admin.py]: Updated!  With the comment added to the model file

**Pull request merged**